src/inventory-service/inventory-acl/src/main/resources/application.properties

```diff
@@ -0,0 +1,23 @@
+quarkus.log.console.format=%d{HH:mm:ss} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId}, sampled=%X{sampled} [%c{2.}] (%t) %s%e%n
+
+# Centralized configuration values
+table.name=${TABLE_NAME:InventoryTable}
```

Comment:
This looks weird to me - I see what you are doing, but _why_? In what case is `InventoryTable` used? Some local cloud-emulated mode? If yes, then i'd add a specific properties file for that case. 

 application properties can be set by environment variable anyway, so in your deployed prod env, you'd just set the appropriate env var, and not explicitly carry it along in the props file.

Reply:
I can imagine using this override mechanism for defaults for things that you _might_ choose to override for whatever reason, but when it's something it needs to run it feels off. I reckon it should be more explicit by using multiple config files for different deployment situations.

Reply:
@scottgerring can we chat about this Monday so  I fully understand?